src/IdentityServer4/src/Configuration/CryptoHelper.cs

```diff
@@ -97,6 +98,42 @@ public static HashAlgorithm GetHashAlgorithmForSigningAlgorithm(string signingAl
             }
         }
 
+        /// <summary>
+        /// Returns the matching named curve for RFC 7518 crv value
+        /// </summary>
+        internal static ECCurve GetCurveFromCrvValue(string crv)
+        {
+            switch (crv)
+            {
+                case JsonWebKeyECTypes.P256:
+                    return ECCurve.NamedCurves.nistP256;
+                case JsonWebKeyECTypes.P384:
+                    return ECCurve.NamedCurves.nistP384;
+                case JsonWebKeyECTypes.P521:
+                    return ECCurve.NamedCurves.nistP521;
+                default:
+                    throw new InvalidOperationException($"Unsupported curve type of {crv}");
+            }
+        }
+
+        /// <summary>
+        /// Return the matching RFC 7518 crv value for curve
+        /// </summary>
+        internal static string GetCrvValueFromCurve(ECCurve curve)
+        {
+            switch (curve.Oid.Value)
+            {
+                case "1.2.840.10045.3.1.7":
```

Comment:
Why not use the CurveOids constants defined below?

Reply:
@scottbrady91 comment?

Reply:
Because I created the constants to replace these, and then promptly forgot about them :)

Reply:
Updated! Thanks @Randy808